core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -98,6 +98,13 @@ oppia.directive('tutorCard', [
             SolutionManagerService.reset(solution);
             $scope.solutionExists = Boolean(solution);
 
+            $rootScope.$broadcast(
```

Comment:
Hm, could we just use the existing EVENT_ACTIVE_CARD_CHANGED? That's the event that the footer is listening for, and presumably all the rest of the info can be obtained from the HintManagerService.

Reply:
@kevintab95 did you see this?

Reply:
Yes, this will be done once the new API is in place.